joomla/joomla-cms
# Fix for nested sets in the menu (installation and update processes)

Please see #8274 and #8057 for more details.

We need to correct the nested sets after we've removed one entry in the menu.

Thanks to @matrikular and @roland-d for providing valued input on this topic.

**How to test?**

Do an installation and an update (with the changed files included in the package) and check whether the menu still works properly!
